Skip customer sync setting for new stores [MAILPOET-4652]

costasovo opened the pull request:
## Description
This PR changes how we display the WooCommerce setup screen. 

The screen for shops with no customers:
![Screenshot 2022-10-07 at 10 39 04](https://user-images.githubusercontent.com/1082140/194511263-05ad18c4-831c-4235-bd1d-0d4a117b1812.png)

The screen for shops with some customers (guest or registered).
![Screenshot 2022-10-07 at 10 38 27](https://user-images.githubusercontent.com/1082140/194511388-a62c4cb8-74e5-4cca-a201-2d9cf40cb50a.png)

Note: During implementation, I found out that we need to save a value for the first checkbox event when it is not displayed. This is because it is then used in `Settings > WooCommerce > Subscribe old WooCommerce customers`, and the setting is used in Customer sync for [updating the status of subscribers who don't have confirmation records](https://github.com/mailpoet/mailpoet/blob/trunk/mailpoet/lib/Segments/WooCommerce.php#L527-L551). I discussed it with @pavel-mailpoet in the Jira ticket, and we chose to use `false` since it is safer for list hygiene.

## Code review notes
I discussed the approach for fetching the subscribers count with the proton team.


## QA notes
The page can be displayed any time by going to URL: `wp-admin/admin.php?page=mailpoet-woocommerce-setup#/woocommerce`

1) Check the screen on with a fresh WooCommerce installation without any customers. You should see only one setting.
2) Create an order or register a customer and check the screen again and you should see both settings.

## Linked PRs

_N/A_

## Linked tickets

[MAILPOET-4652]

## After-merge notes

_N/A_


[MAILPOET-4652]: https://mailpoet.atlassian.net/browse/MAILPOET-4652?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

costasovo committed:
Unify default values of mailpoet_subscribe_old_woocommerce_customers

The value is used in case the setting is not defined. Normally this shouldn't
happen because we set the value in the wizard.
The default value used in PHP is false, but in settings in admin was true.
See https://github.com/mailpoet/mailpoet/blob/b0aacdd4ef3b0cff70046c01325d11d5c1aa53ed/mailpoet/lib/Segments/WooCommerce.php#L528
This commit unifies it.

[MAILPOET-4652]

costasovo committed:
Add method for getting customers count to WooCommerce helper

[MAILPOET-4652]

costasovo committed:
Hide WooCommerce initial import from wizard when there are no customers

[MAILPOET-4672]

costasovo committed:
Update WooCommerce cookie tracking info in Wizard

[MAILPOET-4652]

costasovo committed:
Update acceptance tests for Woo setup page

[MAILPOET-4652]

rodrigoprimo reviewed (dismissed):
Looks good to me.

Aschepikov reviewed (commented):
Hi, @costasovo there is one issue with woocommerce settings in the 4s step in the wizard process.
In both cases (with woocommerce customers and without them in the 4s step I see just one setting, but the button ¨start using mailpoet¨ is frozen. 
In the 1 case (without wc customers) I can go to another page of the mailpoet and there I will see 1 setting for 4s step with working button.

https://user-images.githubusercontent.com/113185437/196198263-2db684a8-ec72-486c-bd23-3bedad286b85.mp4

In the 2 case (with wc customers) I can go to another page of the mailpoet too and there I will see 2 settings for 4s step with working button.

![image](https://user-images.githubusercontent.com/113185437/196198538-a639f593-074f-4206-ab9a-4c5eeb9ed6cc.png)

https://user-images.githubusercontent.com/113185437/196198322-1e92747a-c7ea-41e6-8620-71b54a2932b0.mp4

costasovo committed:
Fix WooCommerce setup in the welcome wizard

The component that requires the global mailpoet_show_customers_import
variable is used also on the wizard page.
This commit ensures that it is set on the page.
[MAILPOET-4652]

costasovo commented:
@JanJakes I added one new commit. I forgot to add the JS global variable to the wizard page.
@Aschepikov Good catch! I was testing only on the `mailpoet-woocommerce-setup` page and forgot to update data on the Wizard page. Let's wait for the code review of changes, and then please, you can check again.

Aschepikov reviewed (approved):
Verified, GJ!
